include/curl/curl.h

```diff
@@ -1314,9 +1316,9 @@ typedef enum {
      Note that setting multiple bits may cause extra network round-trips. */
   CINIT(HTTPAUTH, LONG, 107),
 
-  /* Set the ssl context callback function, currently only for OpenSSL ssl_ctx
-     in second argument. The function must be matching the
-     curl_ssl_ctx_callback proto. */
+  /* Set the ssl context callback function, currently only for OpenSSL or
+     WolfSSL ssl_ctx or mbedTLS mbedtls_ssl_config in second argument.
```

Comment:
I would insert the old "oxford comma" between `ssl_ctx` and `or`, and a `the` between `in` and `second`.

Reply:
Yes, good point (or comma!)